src/main/template/main.handlebars

```diff
@@ -11,16 +11,35 @@
                 <div class="scope-selector">
                     <div class="area controls">
                         <label for="input_baseUrl">Base URL</label>
-                        <span id="input_baseUrl" style="word-wrap: break-word;font-size:1.2rem;margin-bottom: 17px;display: block;" >{{baseUrl}}</span>
+                        <div class="input-group" style="margin-bottom: 0.5rem;">
+                            <span id="input_baseUrl">{{baseUrl}}</span>
+                        </div>
 
-                        <label for="input_apiKey">Access Token (JWT)</label>
+                        <label for="input_apiKey">Bearer Token (JWT)</label>
```

Comment:
Great question, I realize that this little word change is a big part of the change for NG-5085. My intent with this change was to keep the letters "JWT" in the label (because people seem to find that useful here) but to diffuse the possibility for confusion about what kind of token appears in the field. 

Why did it ever say "Access Token"? Well, when we started using this project the recommended practice we were telling people was to obtain an `id_token` from Auth0 for the portal client_id, use it to obtain a delegated `id_token` from Auth0 for the target client_id, and then send that as a Bearer token. Therefore we went with the first phrase that kinda made sense, introduced the idea of a JWT, and didn't tell any lies about being sent via an `Authorization: $TYPE $TOKEN` header. ("Bearer" is a token type, of course.)

However, our recommendations have better and more sophisticated. One nuance is that some people now use Auth0 `access_token`s and also `id_token`s. Regardless of whether the value that the portal puts in this input is a JWT or not, and if it's an `id_token` or an `access_token`, I think that the plainest way to name the thing we're asking for is simply "Bearer token". 

Does this make sense?